### DISCUSSION TITLE
Custom Allow Message

### DISCUSSION THREAD

**Discussion by ApexAlphaHero** [farling42/fvtt-easy-regions#6] [Ideas]
Instead of the static allow message of "Do you want to teleport *token name* to *region name* on *scene name.* I'd like the option to configure a custom allow message so it could say something like "Do you want *token name* to take the stairs down?"

**Comment by farling42** [comment]
The string is embedded deep within the core Foundry code at `client-esm\data\region-behaviors\teleport-token.mjs`

It also uses private functions, so there's no easy way to override the behavior without completely copying it and changing that one line.

**Comment by farling42** [comment]
See https://github.com/foundryvtt/foundryvtt/issues/11470 for the request made for core Foundry behavior changes.

**Reply by farling42** [reply]
I've added this function in the latest release of Easy Regions.
